release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -2832,6 +2825,107 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+* Previously with oc-mirror plugin v2 (Technology Preview), images that referenced both `tag` and `digest` references were not supported. During the disk-to-mirror process for fully disconnected environments, the images were skipped and this caused build issues for the archive file. With this release, oc-mirror plugin v2 now supports an image that includes both these references. An image is now pulled from the `digest` reference and keeps the `tag` reference for information purposes,  while an appropriate warning message is displayed in the console output. (link:https://issues.redhat.com/browse/OCPBUGS-42421[*OCPBUGS-42421*])
+
+* Previously, the Cluster API used an unsupported tag template for a virtual network installation when compared with the default non-cluster-API-provisioned installation. This caused the Image Registry Operator to enter a degraded state when configured with `networkAccess: Internal`. With this release, the Image Registry Operator now supports both tag templates so that this issue no longer exists. (link:https://issues.redhat.com/browse/OCPBUGS-42394[*OCPBUGS-42394*])
+
+* Previously, the Cloud Controller Manager (CCM) liveness probe used on {ibm-cloud-title} cluster installations could not use loopback and this caused the probe to continously restart. With this release, the probe can use loopback so that this issue not longer occurs. (link:https://issues.redhat.com/browse/OCPBUGS-41941[*OCPBUGS-41941*])
+
+* Previously, when the `globallyDisableIrqLoadBalancing` field was set to `true` in the `PerformanceProfile` object, the isolated CPUs were listed in the `IRQBALANCE_BANNED_CPULIST` variable instead of the `IRQBALANCE_BANNED_CPUS` variable. Changing the value of the `globallyDisableIrqLoadBalancing` field from `true` to `false` did not update the `IRQBALANCE_BANNED_CPULIST` variable correctly. As a result, the number of CPUs available for load rebalancing did not increase because the isolated CPUs remained in the `IRQBALANCE_BANNED_CPULIST` variable. With this release, a fix ensures that isolated CPUs are now listed in the `IRQBALANCE_BANNED_CPUS` variable, so that the number of CPUs available for load rebalancing increase as expected. (link:https://issues.redhat.com/browse/OCPBUGS-42323[*OCPBUGS-42323*])
+// Remove the OCPBUGS-42323 known issue in 4.17.0? Martin Sivak is the contact as Yanir is travelling.
+
+// OCPBUGS-39121 is a known issue? Martin K to confirm.
+
+[id="ocp-4-17-1-updating_{context}"]
+==== Updating
+To update an {product-title} 4.16 cluster to this latest release, see xref:../updating/updating_a_cluster/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
```

Comment:
```suggestion
To update an {product-title} 4.16 cluster to this latest release, see xref:../updating/updating_a_cluster/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster by using the CLI].
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#u

Reply:
I'll retain as is because the linked section states "Updating a cluster by using the CLI"